infra/versions.tf

```
@@ -19,12 +19,12 @@ terraform {
   required_providers {
     google = {
       source  = "hashicorp/google"
-      version = ">= 3.53, < 4.84.1"
+      version = ">= 3.53, < 6"
```

IIRC `4.83.0` was the first to support `replication { auto {} }` for `google_secret_manager_secret `
```suggestion
      version = ">= 4.83.0, < 6"
```